### DISCUSSION TITLE
Can v-navigation-drawer be pinned to container?

### DISCUSSION THREAD

**Discussion by windbridges** [vuetifyjs/vuetify#12247] [General]
Hello! I want to build one-column layout having maximum width = 1200px. Also it should have a permanent navigation drawer on the right side. On narrow screen sizes the conent area can shrink. But when screen is large, content can not be larger than 1200px. Space should be added from left and right sides in that case.

My problem is that when I set max-width to the content, the navigation drawer is always pinned to the right side of screen, while content area is centered. Is it possible to pin navigation drawer to right side of the content instead of the right side of screen?

**Comment by mtermoul** [comment]
Did you try Discord support channel?
https://discord.com/invite/s93b7Fv
I have seen many issues resolved by team members on the help channel.
